test/test_torch.py

```diff
@@ -6138,12 +6147,68 @@ def test_inverse_helper_zero_size(size):
             return
 
         from numpy.linalg import inv
-        matrices = random_fullrank_matrix_distinct_singular_value(3, 2).to(device).permute(0, 2, 1)
+        matrices = random_fullrank_matrix_distinct_singular_value(3, 2, dtype=dtype).to(device).permute(0, 2, 1)
         assert not matrices.is_contiguous()
         matrices_inverse = torch.inverse(matrices)
         expected_inv = torch.as_tensor(inv(matrices.cpu().numpy()))
         self.assertEqual(matrices_inverse, expected_inv.to(device))
 
+    # TODO: once there is more support for complex dtype, they shall be added to above test
+    @unittest.expectedFailure
+    @onlyCUDA
+    @skipCUDAIf(
+        _get_torch_cuda_version() < [10, 0] and not TEST_MAGMA,
+        "On cuda 9.2, torch.inverse relies on magma"
+    )
+    @dtypes(torch.complex64, torch.complex128)
+    def test_inverse_complex_xfailed(self, device, dtype):
+        from torch.testing._internal.common_utils import random_fullrank_matrix_distinct_singular_value
+
+        def test_inverse_helper(matrix, batches, n):
+            identity = torch.eye(n, dtype=dtype, device=device)
+
+            # correctness test, check matrix*matrix_inverse == identity
+            matrix_inverse = torch.inverse(matrix)
+
+            self.assertEqual(identity.expand_as(matrix), torch.matmul(matrix, matrix_inverse))
```

Comment:
so you are xfailing this because batched matmul does not work? Can you do a loop over batch for now, so that complex inverse is tested? 
Ideally, you should add test cases from here and the next test to `test_inverse` and specialize matmul checks for complex. 
Is anyone working on enabling bmm on complex? Does not seem like it should be hard to do.

Reply:
There is an open PR https://github.com/pytorch/pytorch/pull/42553 that enables bmm for complex.